content/posts/2021/01/2021-11-04-Getting-Started-With-React/index.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Gettting Started with React
+author: Valary Uhuru
+date: 2021-11-04
+hero: ./images/image.png
```

Comment:
Hero image not found.

You need a hero image that will be previewed when the blogs list is fetched.

In the screenshot, your hero image shows up empty.  [Read more on this](https://womenintechblog.dev/How-to-add-a-blog) 
![Screenshot 2021-11-28 at 22 50 25](https://user-images.githubusercontent.com/9129843/143783676-16ff5f70-e725-4d41-9165-cee5a1cd748c.png)